### DISCUSSION TITLE
GitHub Copilot

### DISCUSSION THREAD

**Discussion by anilAspire** [community/community#131628] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

does Github expose an api for GitHub copilot

**Comment by Akash1134** [comment, marked as answer]
Hi @anilAspire 

Copilot is integrated into various IDEs through extensions and plugins. These plugins interact with the Copilot service to provide code suggestions based on the context of the code being written. GitHub does not expose a public API specifically for Copilot. Please refer this [FAQ](https://resources.github.com/copilot-trust-center/) for details.

Thanks!
